packages/markers/src/browser/problem/problem-tree-model.ts

```diff
@@ -20,15 +20,53 @@ import { MarkerNode, MarkerTree, MarkerOptions, MarkerInfoNode } from '../marker
 import { MarkerTreeModel } from '../marker-tree-model';
 import { injectable, inject } from 'inversify';
 import { OpenerOptions, TreeNode } from '@theia/core/lib/browser';
+import { Marker } from '../../common/marker';
 import { Diagnostic } from 'vscode-languageserver-types';
+import { ProblemUtils } from './problem-utils';
 
 @injectable()
 export class ProblemTree extends MarkerTree<Diagnostic> {
+
     constructor(
         @inject(ProblemManager) protected readonly problemManager: ProblemManager,
         @inject(MarkerOptions) protected readonly markerOptions: MarkerOptions) {
         super(problemManager, markerOptions);
     }
+
+    protected getMarkerNodes(parent: MarkerInfoNode, markers: Marker<Diagnostic>[]): MarkerNode[] {
+        return markers.map((marker, index) => this.createMarkerNode(marker, index, parent)).sort((a, b) => this.sortMarkers(a, b));
```

Comment:
`return super.getMarkerNodes(parent, markers).sort((a, b) => this.sortMarkers(a, b));`

Not calling a super implementation a common cause fo loosing functionality later.